CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 1.27.0 - 5 October 2023
+### Added
+- Enhanced registries functionality: (https://github.com/microsoft/vscode-docker/issues/869)
```

Comment:
This applies to all of these links plus the ones from 1.26.1, but the style is different. The earlier style is `[#1234](https://github.com/microsoft/vscode-docker/issues/1234)`, but it changed to `(https://github.com/microsoft/vscode-docker/issues/1234)`. We should change the current milestone and 1.26.1 to match the old style.

Reply:
ahh you're right, I just realized haha